CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### 2.3.1 (TBD)
 
+- Feature: Agents can now be installed using a mutator webhook
+- Change: The `trafficManagerConnect` timout default has changed from 20 seconds to 60 seconds, in order to facilitate TODO.
```

Comment:
So I didn't actually finish writing that sentence.

Reply:
I'll replace the `TODO` with `the extended time it takes to apply everything needed for the mutator webhook.`